### ISSUE TITLE
can't access the "overwrite" menu

### ISSUE THREAD

**Issue by NUMER013** [iLollek/NRPFarMod#17]
when i press "insert", the mouse doesn't appear, so it's impossible to activate overwrite mode

**Comment by iLollek** [comment]
hello, I'm going to test this soon.

You can still access overwrite mode using the hotkeys though. Please refer to the documentation for more information.

**Comment by NUMER013** [comment]
Yes, hotkeys work thought, but only if I don't activate playlist mode in the first place, in which case the logs say “Unable to use Overwrite-mode: Playlist-mode is enabled!”

**Comment by iLollek** [comment]
As the documentation says, this behavior is expected. You are not able to use Overwrite mode if playlist mode is already enabled. 

You also should not be able to use overwrite mode via the HUD (GUI) when playlist mode is enabled. This has nothing to do with your Issue though; I tested this and you're correct, the mouse doesn't appear when you open the HUD/GUI using the INSERT key.

**Comment by NUMER013** [comment]
Oh, I see, but what I really meant was that there's no way to go back to overwrite mode from playlist mode, unless I restart my game. Maybe this will indeed be fixed with the mouse issue but otherwise it would be cool to have a feature to quit playlist mode to go back to overwrite mode (I swear I've read the documentation I just want to help lol)

**Comment by iLollek** [comment]
No Problem about the docu thing -- im sure you read it, you're one of the only people that actually report bugs as tickets, and im more than thankful for that. you're a big help to the project!!

heh, funny you did actually find a bug i was missing out on by actually trying out a feature that's missing... yeah, you're right, i missed that; you cant deactivate playlist mode without restarting the game! I will create a seperate issue for this. This one is for the mouse not appearing.

**Comment by NUMER013** [comment]
Always a pleasure to help, the mod alone literally made me put hours back into the game haha it's so good! thanks for your time and patience

**Comment by iLollek** [comment]
Happy to hear that. Looking forward to release the new version soon.
